.mailmap

```diff
@@ -0,0 +1,16 @@
+Ary Borenszweig <asterite@gmail.com> <aborenszweig@manas.com.ar>
+Ary Borenszweig <asterite@gmail.com> <ary@esperanto.org.ar>
+Ary Borenszweig <asterite@gmail.com> <asterite@gmail.com>
+Brian J. Cardiff <bcardiff@gmail.com> <bcardiff@manas.com.ar>
+Brian J. Cardiff <bcardiff@gmail.com> <bcardiff@manas.tech>
+George Dietrich <george@dietrich.app> <yomoejoe@gmail.com>
```

Comment:
Is there a good way to handle me not using either one of these anymore? :sweat_smile:.

Reply:
Sure, we can put down any address as merge target.

Reply:
`dev@dietrich.pub` would be great :pray:.